doc/adr/0005-use-elastic-search-for-data-storage.md

```diff
@@ -0,0 +1,22 @@
+# 5. Use Elastic Search for Data Storage
+
+Date: 2017-08-24
+
+## Status
+
+Accepted
+
+## Context
+
+The same GP data is held in MongoDB and Elasticsearch.
+Elasticsearch is configured as a cluster for reliability and failover, and provides a single point for data updates.
+MongoDB runs as a single instance and is not clustered.
+
+## Decision
+
+To reduce the deployed components Profiles will consume data from Elasticsearch rather than MongoDB, ensuring that the GP-Finder and Profiles use the same data source.
```

Comment:
The reduction of deployed components is a consequence rather than the decision. Suggest the sentence is cut down to just `Profiles will consume data from Elasticsearch rather than MongoDB.` and move the start and end points into the consequences.